collectors/pod_test.go

```diff
@@ -261,18 +261,90 @@ func TestPodCollector(t *testing.T) {
 						},
 					},
 				},
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "pod3",
+						Namespace: "ns3",
+					},
+					Status: v1.PodStatus{
+						ContainerStatuses: []v1.ContainerStatus{
+							v1.ContainerStatus{
+								Name: "container1",
+								State: v1.ContainerState{
+									Terminated: &v1.ContainerStateTerminated{
+										Reason: "CrashLoopBackOff",
```

Comment:
`CrashLoopBackOff` should be `ContainerStateWaiting` instead of `ContainerStateTerminated`.